JASP-Engine/JASP/R/bainregressionlinearbayesian.R

```diff
@@ -141,16 +144,23 @@ BainRegressionLinearBayesian <- function(jaspResults, dataset, options, ...) {
 
 	bainResult <- bainContainer[["bainResult"]]$object
 	sum_model <- summary(bainResult)
-
+	
 	groups <- as.character(sum_model[["Parameter"]])
-	estim <- sum_model[["Estimate"]]
+	N <- sum_model[["n"]]
+	mu <- sum_model[["Estimate"]]
 	CiLower <- sum_model[["lb"]]
 	CiUpper <- sum_model[["ub"]]
-	SE <- (CiUpper - CiLower)/(2 * qnorm(0.975))
 
-	d <- data.frame(v = groups, mean = estim, SE = SE, CiLower = CiLower, CiUpper = CiUpper)
+	# Deduct standard error from 95 percent confidence interval
```

Comment:
Did you mean `Deduce` here? That comment threw me off a little :p

Reply:
Hehe you are right... I will change that